frontend/src/bundles/common/components/slider/slider.tsx

```diff
@@ -0,0 +1,64 @@
+import { Box, Slider } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { useCallback } from 'react';
+
+import { getValidClassNames } from '../../helpers/helpers.js';
+import { defaultExperienceMarks } from './default-values.js';
+import classes from './style.module.css';
+import { defaultSliderStyle } from './styles.js';
+
+type Mark = {
+    label?: string;
+    value: number;
+};
+
+type Properties = {
+    label?: string;
+    value?: number;
+    className?: string;
+    step?: number | null;
+    valueLabelDisplay?: 'auto' | 'on' | 'off';
+    marks: Mark[];
+};
+
+const CustomSlider: React.FC<Properties> = ({
+    label,
+    value,
+    marks = defaultExperienceMarks,
```

Comment:
I think you can delete defaultExperienceMarks or make it like constant for all project if this array used in different part of application, because in this component we create general slider component without any context, so just create constant for variable defaultExperienceMarks, if it needed and move it to general folder, only if we will use it in whole app. After that provide this value as props, but default value for this variable should be empty array.
Also about name, name it please like options or something like that, more general without context.

Reply:
ok, I will just delete it